shell/platform/common/engine_switches_unittests.cc

```diff
@@ -83,6 +86,30 @@ TEST(FlutterProjectBundle, SwitchesMissingValues) {
   // switch in the array.
   EXPECT_EQ(switches[2], "--oops");
 }
+
+TEST(EngineSwitches, SkiaTraceAllowlist) {
+  fml::CommandLine command_line =
+      fml::CommandLineFromInitializerList({"command"});
+  Settings settings = SettingsFromCommandLine(command_line);
+  EXPECT_EQ(settings.trace_skia, true);
```

Comment:
EXPECT_TRUE? (here and below)

Also, if it is always true, why do we have it any more as a settings value?

Reply:
Done

`settings.trace_skia` will be false in release builds